### PR TITLE
Provide a Dockerfile for the bridge

### PR DESCRIPTION
**Pull request by kaiyou**
This PR provides a first implementation of a Dockerfile for this bridge.

It replaces #702.

### CODE REVIEW

**Comment by kaiyou** [comment]
Still missing some documentation, I hope I can provide something in the next few days.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+RUN git clone https://github.com/matrix-org/freebindfree.git \
+ && cd freebindfree \
+ && make
+
+COPY ./package.json ./package.json
+RUN npm install
+
+# App
+FROM node:6-slim
```
Comment (Half-Shot) [inline Dockerfile]: Still not sure why we're targetting node 6? Node 10 is definitely going to yield better perf

**Comment by simonszu** [comment]
Thank you so much for your Dockerfile. However, since it is a little unclear for me how to use the container, i want to note that a modified Readme would look good in this PR as well ;)

**Comment by xNephe** [comment]
Thank you for the dockerfile, it's really useful !
However, I see no volume definition for the databases and the configuration files. From my understanding, `/config` and `/app/data` should be declared as volumes.

**Pull request merged**